### PR TITLE
Refactor(jira): Extract a separate sub project for Jira.

### PR DESCRIPTION
**Pull request by psantos1113**
Move the Jira Issue Tracker code into it's own project so it can be extracted as a library. No logic changes just updates to build and package structure.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -79,6 +79,7 @@ dependencies {
     implementation project(':alert-common')
     implementation project(':alert-database')
     implementation project(':alert-issuetracker')
+    implementation project(':alert-issuetracker-jira')
```
Comment (bamandel) [inline build.gradle]: is the alert-issuetracker import still necessary if we're pulling in the jira version?

File: alert-issuetracker-jira/src/main/java/com/synopsys/integration/issuetracker/jira/cloud/JiraCloudContext.java [hunk 2]
```diff
@@ -20,7 +20,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package com.synopsys.integration.alert.issuetracker.jira.cloud;
```
Comment (gkillough) [inline alert-issuetracker-jira/src/main/java/com/synopsys/integration/issuetracker/jira/cloud/JiraCloudContext.java]: If these are still in alert, should we keep that in the package name?

Comment (psantos1113) [reply alert-issuetracker-jira/src/main/java/com/synopsys/integration/issuetracker/jira/cloud/JiraCloudContext.java]: I'm looking to move the sub projects into their own repositories this week.  That's the reason for the package change there.

File: build.gradle (hunk 1 above)
Comment (psantos1113) [reply build.gradle]: yes I should be able just include alert-issuetracker-jira and get the issuetracker project as a transitive.

**Pull request merged**